forc/src/utils/cli_error.rs

```diff
@@ -38,6 +38,11 @@ impl CliError {
         );
         Self { message }
     }
+
+    pub fn fuel_core_not_running(node_url: &str) -> Self {
+        let message = format!("could not get a response from node at the URL {}. Start a node with `fuel-core`. See <https://github.com/FuelLabs/fuel-core#running> for more information", node_url);
```

Comment:
I'm surprised `rustfmt` doesn't try and wrap this or break the `format!` args onto different lines, but if rustfmt's happy I'm happy!